Makefile

```diff
@@ -15,16 +15,16 @@ test-harness: ## Run harness tests
 test-harness-ci: build
 	MOCK_ADDRESS=127.0.0.1:8080 NETWORK_SIZE=3 DUSK_BLOCKCHAIN=${PWD}/bin/dusk DUSK_UTILS=${PWD}/bin/utils DUSK_SEEDER=${PWD}/bin/voucher DUSK_WALLET_PASS="password" make test-harness
 test-harness-alive: stop build
-	MOCK_ADDRESS=127.0.0.1:9191 NETWORK_SIZE=3 DUSK_BLOCKCHAIN=${PWD}/bin/dusk DUSK_UTILS=${PWD}/bin/utils DUSK_SEEDER=${PWD}/bin/voucher DUSK_WALLET_PASS="password" \
-	go test -v --count=1 --test.timeout=0 ./harness/tests/ -args -enable -keepalive
+	MOCK_ADDRESS=127.0.0.1:9191 NETWORK_SIZE=9 DUSK_BLOCKCHAIN=${PWD}/bin/dusk DUSK_UTILS=${PWD}/bin/utils DUSK_SEEDER=${PWD}/bin/voucher DUSK_WALLET_PASS="password" \
+	go test -v --count=1 --test.timeout=0 ./harness/tests/ -run TestMultipleBiddersProvisioners -args -enable -keepalive
```

Comment:
Is the `run TestMultipleBiddersProvisioners` really required here or this was left over accidentally ?

Reply:
Not accidentally. The suggestion here is to use this `TestMultipleBiddersProvisioners` test for any long-run (keepalive-mode) testing. `TestMultipleBiddersProvisioners` should be maintained and extended later on in order to provide the most complex and realistic network scenarios.